kyo-bench/src/main/scala/kyo/bench/MtlBench.scala

```diff
@@ -18,14 +18,14 @@ class MtlBench extends Bench:
                 for
                     conf <- Envs.use[Env](_.config)
                     _    <- Sums.add(Event(s"Env = $conf"))
-                    _    <- Vars[State].update(state => state.copy(value = state.value + 1))
+                    _    <- Vars.update[State](state => state.copy(value = state.value + 1))
```

Comment:
Can you try adding the type parameter to the lambda instead of the 'update'?

Reply:
Yep, that works too.